tests/test_model.py

```diff
@@ -10,6 +11,45 @@
 from mlserver.codecs.pandas import PandasCodec
 from mlserver.model import MLModel
 
+from .fixtures import TextModel, TextStreamModel
+
+
+async def stream_generator(generate_request):
+    yield generate_request
+
+
+async def test_predict_stream_fallback(
```

Comment:
as explained earlier I am not sure if we should fallback to `predict` or raise not implemented.